chai-immutable.js

```diff
@@ -71,7 +71,7 @@ module.exports = function (chai, utils) {
     return function (collection) {
       var obj = this._obj;
 
-      if (obj && obj instanceof Collection) {
+      if (obj && obj instanceof Collection || collection instanceof Collection) {
```

Comment:
Hm, yeah it doesn't seem like we do…sorry, I can't remember anymore! I can remove it and see if we stumble on why again. Sound good?